moped-editor/src/views/projects/projectView/ProjectActivityLogTableMaps.js

```diff
@@ -528,7 +528,7 @@ export const ProjectActivityLogTableMaps = {
       },
     },
   },
-  moped_proj_contract: {
+  moped_proj_work_activity: {
```

Comment:
I want to request another change, but it would be totally fine to leave it out of scope for now.

We now have the special generated column `reference_id` which would be nice to use to identify the activities in the log. 

So this:

```
Edited the contract for MASTEC ZAPALAC-REED DEVELOPER by updating the task orders
```

becomes:

```
Edited the work activity MPD-1661-156 by updating the task orders
```

In some of the ATSD examples they will have multiple work activities that use the same workgroup/contractor, so it will be best to use the reference ID.